### ISSUE TITLE
twamp-responder doesnt run on Windows7 Pro Service Pack 1

### ISSUE THREAD

**Issue by jcsv07** [demirten/twamp-gui#14]
Hi there

I installed "TwampGuiInstaller-1.0.5" on Win 7 Pro SP1 but when I open twamp-responder it doesn't work properly, the windows doesn't show any information and just appear the border of the window but not buttons, logs or any other info. I have tried in others computer and it's work, so I suspect that can be a specific configuration on Windows (maybe), unfortunately I can not use the computer in which its works; and more than that, if great to know the issue in order to help to anothers.

![twamp-responder](https://user-images.githubusercontent.com/47420998/52414867-d1493e80-2abb-11e9-8fbc-1febfd80c32b.PNG)

**Comment by tlrsiap** [comment]
same issue I have 
It works on a machine and another one it doesn't , same screenshot , windows of client and server opens but with noting on them

**Comment by duddly-github** [comment]
same issue - trying to run the responder on windows 2019 server - both windows (client and responder) are blank.

**Comment by duddly-github** [comment]
Thanks to Murat Demirten for assisting with this!
He recommended that the issue was with QT QML /Windows - i began looking for solutions to QT/QML apps with a blank screen and found that it is/was (for me) related to video capabilities.  
In MY CASE - I was trying to run the program on a VMWare VM where I had not enabled 3D graphics support.  As soon as I enabled that and rebooted, the TWAMP client and responder worked fully.   I also had to do a little firewall work once I got the GUI working but that was unrelated.

TLDR - if you get the blank screen when running the Windows version, make sure you have 3d Graphics support for the QT QML libraries.  You may be able to run dxdiag.exe to get more info about your capablities.

**Comment by demirten** [comment]
@duddly-github, could you test it again with latest 1.0.7 release?

**Comment by duddly-github** [comment]
it works great!  Version 1.0.7 does not have the "blank dialog" issue that v 1.0.5 did on windows machines without 3D graphics capabilities.  I am running the twamp-responder on Windows Server 2016 in VMWare ESXi 6.7 and I no longer need to enable 3D graphics support to get it to run.  Both the client and the responder work fine.  Thanks!

**Comment by demirten** [comment]
Fixed on v1.0.7
